editor/project_settings_editor.cpp

```diff
@@ -119,7 +120,8 @@ void ProjectSettingsEditor::_notification(int p_what) {
 		case EditorSettings::NOTIFICATION_EDITOR_SETTINGS_CHANGED: {
 
 			search_button->set_icon(get_icon("Search", "EditorIcons"));
-			clear_button->set_icon(get_icon("Close", "EditorIcons"));
+			search_box->add_icon_override("right_icon", get_icon("Search", "EditorIcons"));
```

Comment:
This is not needed, you can override the icon directly here in the editor theme:

https://github.com/godotengine/godot/blob/master/editor/editor_themes.cpp#L827

Reply:
Wouldn't that override the icon for all LineEdit's? Here I implemented it the same way to how it was done previously in other places.

Reply:
Yes, it is correct to override the icon for all line edits, since it's part of the editor theme (which overrides most of the default theme). If you want to enable clearing, just enable it via function.

Reply:
I did it like this because that's how it was done previously. (For example [here](https://github.com/godotengine/godot/blob/master/editor/editor_help.cpp#L256)). Anyway, as requested I've changed all these calls to `set_right_icon function` which doesn't use theme.